docs/api-designer/getting-started-api-designer/README.adoc

```diff
@@ -135,40 +134,41 @@ image::create-api-designer-design.png[Image of page to create an API design]
 . Select the design title and click the pen icon. Change the design title to *Orders* and click the *Save changes* icon.
 . Click the *Design* tab, and optionally edit the design as follows:
 .. Provide a version number and a description.
-.. AsyncAPI only: Define terms of service.
+.. _AsyncAPI only:_ Define terms of service.
 .. Add your contact information (name, email address, and URL).
 .. Select a license.
-.. OpenAPI only: Define tags.
+.. _OpenAPI only:_ Define tags.
 .. Define one or more servers.
 .. Configure a security scheme.
-.. OpenAPI only: Specify security requirements.
-.. OpenAPI only: Configure vendor extensions.
+.. _OpenAPI only:_ Specify security requirements.
+.. _OpenAPI only:_ Configure vendor extensions.
 +
 [.screencapture]
 .Editing an API design
 image::api-designer-editor.png[Image of API design editor]
 +
 . Click the *Source* tab to see a live preview of the design.
```

Comment:
I think it's better to start all of the non-optional steps with a verb. In this case, the "to see a live preview" text is showing the result of the action, not a condition.
The format of the optional step is different. In that case, as you rightly suggest, the "to see the changes" text is a condition to indicate to the user why they might want to do the step. 

Having said all that, I've rephrased to avoid the confusion: 
"Click the *Source* tab, and review the live preview of the design."